Task3/Task3_Atanas_Kisyov/src/python/car_wash/db_connection/connection_pools.py

```diff
@@ -0,0 +1,101 @@
+"""Read-only and read/write database connection pools definitions"""
+from collections import deque
+import logging
+import time
+
+import psycopg2
+
+from python.car_wash.db_connection.pool_base_class import PoolBaseClass
+
+
```

Comment:
Create constant as i showed https://github.com/ulyktey/car_wash_db_pool/blob/main/src/python/car_wash/db/dbpool.py#L10 and use it for class parameters

Reply:
Done.